python_modules/dagster/setup.py

```diff
@@ -104,7 +104,7 @@ def get_version() -> str:
         # https://github.com/mhammond/pywin32/issues/1439
         'pywin32!=226; platform_system=="Windows"',
         "docstring-parser",
-        "universal_pathlib",
+        "universal_pathlib>=0.1.4",
```

Comment:
Can you explain this change? Is it required for the integration?

Reply:
the new version provides useful and stable `@property`s such as `.fs` and `.storage_options` and is backwards compatible with previous versions. I am using these properties in my code. Should also do it in the `UPathIOManager`.

Reply:
moved this version specification to `dagster-polars` lib